wherehows-web/package.json

```diff
@@ -64,23 +64,24 @@
     "ember-concurrency": "^0.8.15",
     "ember-decorators": "^2.0.0",
     "ember-export-application-global": "^2.0.0",
-    "ember-fetch": "^5.1.1",
+    "ember-fetch": "5.1.1",
```

Comment:
Why are we pinning @5.1.1, also if we're are pinning for similar reasons, then why not pin mirage @0.4.1 until [1379](https://github.com/samselikoff/ember-cli-mirage/issues/1379) is resolved?

Reply:
I pinned it down due to having two versions of the same library. Ember-check-dependencies gave me the error. Maybe there is another way to handle it? Is there a yarn command to lock it in the yarn.lock?

Reply:
Hmm... what is importing the conflicting version? can we narrow it down to one?
otherwise there is a [config](https://github.com/linkedin/WhereHows/blob/master/wherehows-web/config/dependency-lint.js) file for dependency lint where you can specify allowed versions

Reply:
`ember-simple-auth`